sale_order_type/models/account_move.py

```diff
@@ -12,7 +12,7 @@ class AccountMove(models.Model):
         string="Sale Type",
         compute="_compute_sale_type_id",
         store=True,
-        readonly=True,
+        readonly="state not in ['draft']",
```

Comment:
These attributes should now be in XML instead of .py files and must be evaluated as Booleans. Please refer to the logs of the `pre-commit` for more information.